templates/example/base.html

```diff
@@ -7,7 +8,12 @@
     {% block title %}
     {% endblock title %}
   </title>
+  <link rel="icon" href="{% static 'assets/img/favicon.ico' %}" type="image/x-icon" />
+
   {% block style %}
+  <link rel="stylesheet" href="{% static 'styles/page_styles/base.css' %}" />
+  {% block base_side_style %}
+  {% endblock base_side_style %}
   {% endblock style %}
```

Comment:
❗ `<link rel="stylesheet" href="{% static 'styles/page_styles/base.css' %}" />`을 `{% block style %}` 밖에 작성하면 `{% block base_side_style %}{% endblock base_side_style %}`을 추가하지 않아도 되겠어요!!
```suggestion
  <link rel="stylesheet" href="{% static 'styles/page_styles/base.css' %}" />
  {% block style %}
  {% endblock style %}
```